controllers/activities.js

```diff
@@ -0,0 +1,43 @@
+const activityRouter = require('express').Router()
+const request = require('request')
+const axios = require('axios')
+
+const models = require('../domain/models')
+
+// TODO: check for logged in
+
+// Delete Activity
+activityRouter.delete('/:activityId', async (req, res) => {
+  const activityId = parseInt(req.params.activityId)
+
+  models.Activity.destroy({
+    where: {
+      id: { $eq: activityId }
+    }
+  }).then(rowsDeleted => {
+    if (rowsDeleted === 1) {
+      console.log('Deleted activity with ID', req.params.activityId)
+      res.status(200).send('Deleted')
+    } else {
+      console.log('Did not delete activity with ID', req.params.activityId)
+      res.status(404).send('Not deleted')
+    }
+  })
+})
+
+// Move Activity from Event to Buffer
+activityRouter.get('/:activityId/tobuffer', async (req, res) => {
```

Comment:
I left out /fromevent{eventId}. The old backend had an extra check for checking if the activity was in the event before moving it to buffer.